tools/check_release.py

```diff
@@ -270,7 +300,7 @@ def upgrade_test_repo(test, user, library, ref, repo_path):
         
         os.rename(lib_file, bak_file)
     else:
-        logging.error("!! Error trying to backup lib file prior to updating.")
+        logging.error("!! Failure to backup lib file prior to updating.")
```

Comment:
Maybe drop the `!!` here as well? It's kind of silly when we already do `logging.error`.